ldap/src/main/java/org/springframework/security/ldap/server/ApacheDSContainer.java

```diff
@@ -68,7 +69,9 @@
  * @author Luke Taylor
  * @author Rob Winch
  * @author Gunnar Hillert
+ * @deprecated Use {@link UnboundIdContainer} instead.
```

Comment:
Can you please add a bit more explanation as to why. Perhaps:

"Use {@link UnboundIdContainer} instead because ApacheDS 1.x is no longer supported with no GA version to replace it"

Reply:
I have updated the reason.